lib/Transforms/DataCopy.cpp

```diff
@@ -93,24 +103,28 @@ struct DmaGeneration : public FunctionPass<DmaGeneration> {
   bool runOnBlock(Block *block);
   uint64_t runOnBlock(Block::iterator begin, Block::iterator end);
 
-  bool generateDma(const MemRefRegion &region, Block *block,
-                   Block::iterator begin, Block::iterator end,
-                   uint64_t *sizeInBytes, Block::iterator *nBegin,
-                   Block::iterator *nEnd);
+  bool generateCopy(const MemRefRegion &region, Block *block,
```

Comment:
Can this function be updated to return LogicalResult?

Reply:
Done.